src/KeyVaultBuild.Cli/TemplateArguments.cs

```diff
@@ -13,5 +14,8 @@ public class TemplateArguments
 
         [Option('v', "vault", Required = true, HelpText = "KeyVault instance to upload values to.")]
         public string Vault { get; set; }
+
+        [Option('a', "appsettings", Separator = ',', Required = false, HelpText = "Which appsettings, comma separated, to push into keyvault. Regex formatted")]
+        public IEnumerable<string> AppSettings { get; set; }
```

Comment:
Thoughts on providing defaults here, or should be always specify?
Also update the readme with the new command 😃